javascript/javascript2/week1/exercise1.js

```diff
@@ -0,0 +1,30 @@
+//Find the shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+console.log(getShortestWord(danishWords)); // returns 'ø'
+
+function getShortestWord (words) {
+    return words.reduce((w1, w2) => w1.length < w2.length ? w1 : w2);
```

Comment:
Good use of reduce!
I'd use an if else to make it a bit more readable:
```js
return words.reduce((shortestWord, currentWord) => {
  if(currentWord.length < shortestWord.length) return currentWord
  return shortestWord // since you've already returned from the if-statement, this will work as an else-statement
})
```